sdk/go/auto/cmd_test.go

```diff
@@ -287,11 +287,11 @@ func TestRunCanceled(t *testing.T) {
 
 	go func() {
 		path := filepath.Join(e.RootPath, "ready")
-		for i := 0; i < 100; i++ {
+		for range 60 {
 			if _, err := os.Stat(path); err == nil {
 				break
```

Comment:
Is it worth adding a log here that we found the path?  So we can see if we ever get here if the test fails again?

Reply:
Good idea, added